docs/local_install_skaffold.md

```diff
@@ -0,0 +1,190 @@
+# Local install of Substra using kubernetes and skaffold
+
+This guide will help you run the Substra platform on your machine with a 2 organisations setup.
+
+- [Setup](#setup)
+  - [1. Get the source code](#1-get-the-source-code)
+  - [2. Install Docker and Kubernetes](#2-install-docker-and-kubernetes)
+  - [3. Configure Kubernetes](#3-configure-kubernetes)
+  - [4. Install skaffold](#4-install-skaffold)
+- [Running the platform](#running-the-platform)
+- [Login, password and urls](#login-password-and-urls)
+
+## Setup
+
+### 1. Get the source code
+
+You'll need either download or clone 3 repositories:
+
+#### Download
+
+Download links:
+
+- [hlf-k8s](https://github.com/SubstraFoundation/hlf-k8s/archive/dev.zip)
+- [substra-backend](https://github.com/SubstraFoundation/substra-backend/archive/dev.zip)
+- [substra-frontend](https://github.com/SubstraFoundation/substra-frontend/archive/dev.zip)
```

Comment:
I would say that cloning repositories is enough (and recommended). Do we need to propose multiple solutions?

Reply:
I'll put cloning as the default method and leave download links "just in case".